test/core/GenArt721CoreV3_Integration.test.ts

```diff
@@ -91,7 +91,7 @@ describe("GenArt721CoreV3", async function () {
         await expectRevert(
           this.genArt721Core
             .connect(this.accounts.artist)
-            .updateProjectScriptJSON(this.projectZero, "lorem ipsum"),
+            .updateProjectScript(this.projectZero, 0, "lorem ipsum"),
```

Comment:
nit: i know this should revert anyway, but should this be `updateProjectScriptType` here?

Reply:
good call, actually intended this to be addProjectScript - pushed the update!